html/index.ejs

```diff
@@ -22,33 +22,33 @@
           new pathways towards achieving a deforestation-free economy.
         </p>
 
-        <p class="paragraph -center">Trase is a new kind of transparency platform, harnessing untapped trade data with the best
-          available sustainability information to link production landscapes – and the impacts within them – to specific traders,
-          manufacturers, retailers, and investors worldwide. Trase answers the looming question of who is connected to environmental
-          and social impact, and who can help transition to a deforestation-free economy. Trase was developed through a partnership
-          between the Stockholm Environment Institute, and Global Canopy Programme.
-        </p>
-
         <div class="c-workflow-graphic">
           <ul class="supply-chain-list">
             <li class="supply-chain-item">
-              <span class="description paragraph -center">Where are they produced?</span>
+              <span class="description title -mono-font -center">Where are they produced?</span>
             </li>
             <li class="supply-chain-item">
-              <span class="description paragraph -center">Who trades <br/>in them?</span>
+              <span class="description title -mono-font -center">Who trades <br/>in them?</span>
```

Comment:
Please use the `description` class to make any change you need instead of using `title` and `-mono-font` classes.